polaris-intellisense/README.md

```diff
@@ -0,0 +1,23 @@
+# shopify-dev-intellisense README
```

Comment:
As we continue to build additional vscode plugins, do we imagine them being all contained in this one plugin or having multiple small ones? I ask because the approach for Figma is call the plugin "Polaris for Figma" and put everything in one place until it becomes a problem.

Should we follow a similar approach here?

Reply:
I similarly envisioned having one main polaris vscode plugin that would include any new features we came up with (rather than multiple plugins), so let's go with that!

I'll update this line as it's a holdover from when I copy pasta'd from the original prototype 😆 🍝